locales/en.json

```diff
@@ -301,6 +301,17 @@
     "log_tools_upgrade": "Upgrade system packages",
     "log_tools_shutdown": "Shutdown your server",
     "log_tools_reboot": "Reboot your server",
+    "log_admin_bash_history": "History of commands ran by admin",
+    "log_alternatives": "Change to default programs",
+    "log_auth": "Connection history",
+    "log_btmp": "Failed login attempts",
+    "log_dpkg": "History of installed debian packages",
+    "log_fail2ban": "Banning history",
+    "log_history": "History of installed debian packages with apt",
+    "log_kernel": "Information and events about the linux kernel",
+    "log_messages": "General system activities",
+    "log_root_bash_history": "History of commands ran by root",
+    "log_wtmp": "Login and logout history",
```

Comment:
Apparently there's no `wtmp` configured...?

Reply:
yep i removed it because this file make a big 500 error in moulinette and is not so much relevant